.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
   - source devtools/travis-ci/install.sh
   - export PYTHONUNBUFFERED=true
   # Unpack encrypted OpenEye license file
-  - if [ "$TRAVIS_SECURE_ENV_VARS" == true ]; then openssl aes-256-cbc -K $encrypted_5f58236b8908_key -iv $encrypted_5f58236b8908_iv -in oe_license.txt.enc -out $OE_LICENSE -d; fi
+  - if [ "$TRAVIS_SECURE_ENV_VARS" == true ]; then openssl aes-256-cbc -K $encrypted_4fb296b25e14_key -iv $encrypted_4fb296b25e14_iv -in oe_license.txt.enc -out oe_license.txt -d; fi
```

Comment:
The deleted line line had `-out $OE_LICENSE` and the new line has `-out oe_license.txt` I'll try going back to the `$OE_LICENSE`

Reply:
Ah, yes, possibly I broke it while fixing it.

Reply:
well that didn't do it...